common/supplemental/supplementalData.xml

```diff
@@ -5411,9 +5411,16 @@ XXX Code for transations where no currency is involved
 		<parentLocale parent="en_IN" locales="hi_Latn"/>
 		<parentLocale parent="es_419" locales="es_AR es_BO es_BR es_BZ es_CL es_CO es_CR es_CU es_DO es_EC es_GT es_HN es_MX es_NI es_PA es_PE es_PR es_PY es_SV es_US es_UY es_VE"/>
 		<parentLocale parent="fr_HT" locales="ht"/>
-		<parentLocale parent="no" locales="nb nn"/>
+		<parentLocale parent="no" locales="nb nn no_NO"/>
```

Comment:
This one has a comment in the ICU file:

```
            <!-- This one is a bit silly, it is just to generate a stub for no_NO, which is
                 not in CLDR. If we do not do this, then including it in localeIds will generate
                 empty no_Latn and no_Latn_NO and then no_NO aliasing to no_Latn_NO. -->
```

Reply:
Hmmm. let's bring that comment into the .xml file.